proposals/0448-type-variable-scoping.rst

```diff
@@ -331,8 +446,12 @@ Effects
 Alternatives
 ~~~~~~~~~~~~
 
-1. Previous versions of this proposal, along with the accepted `#285`_, use ``-XNoPatternSignatureBinds`` instead of ``-Wpattern-signature-binds``.
-   However, there seems to be no good reason this must be an extension, instead of a warning.
+1. We could further break down ``-XImplicitBinds``, like before.
+
+   But fixing the drafting error would require a *third* extension, ``-XNegativeSignatureBinds``, in addition to the original two.
+   This would allow more conservative defaults --- we must have Haskell98 implicit foralls but not the others which are all guarded behind language extensions today.
+
+   However, @Ericson2314 sensed there is a weariness with too many extensions coming from this, and so didn't do it.
```

Comment:
```suggestion
1. We could further break down ``-XImplicitBinds``, like before.

   This would allow more conservative defaults --- we must have Haskell98 implicit foralls but not the others which are all guarded behind language extensions today.
   
   But fixing the drafting error in `#285`_, where certain examples meant to be including under `-XPatternSignatureBinds` that wouldn't be because they are not "pattern signatures", in this manner would require a *third* extension, ``-XNegativeSignatureBinds``, in addition to the original two.
   @Ericson2314 sensed there is a weariness with so many extensions coming from this proposal (448) already, and so didn't want to pursue this route.
```